src/Jackett.Common/Indexers/ThePirateBay.cs

```diff
@@ -232,6 +229,8 @@ private ReleaseInfo CreateReleaseInfo(QueryResponseItem item)
                 Author = item.Username,
                 DownloadVolumeFactor = 0,
                 UploadVolumeFactor = 1,
+                MinimumRatio = 1,
+                MinimumSeedTime = 172800, // 48 hours
```

Comment:
Huh?
its a public site so there is no requirement to seed and no hit and run penalties.
Generally only private sites (and some few semi private ones) that have these.
I do not believe public sites should have MR and MST in their indexers.

Reply:
All public trackers have those values in Jackett. We can remove them all public trackers if you want to.

Reply:
Yes, public trackers should either not have MR and MST or perhaps they should be set to zero. (I favour not having any).
None of the Cardigann public indexers have MR and MST unless they were once private and became public and someone forgot to remove them.